lib/cancan/conditions_matcher.rb

```diff
@@ -82,7 +82,12 @@ def matches_hash_conditions(adapter, subject, conditions)
       end
     end
 
-    def condition_match?(attribute, value)
+    def condition_match?(attribute, value) # rubocop:disable Metrics/MethodLength
```

Comment:
Not sure how to get around the offence without drastically refactoring the method, which seems a lot out of scope for this PR? :-)